### ISSUE TITLE
ImageMagick usage

### ISSUE THREAD

**Issue by saveryanov** [StephanGeorg/staticmaps#5]
Hi! 

gm module supports ImageMagick usage instead of GraphicsMagick in such a simple way: 
https://www.npmjs.com/package/gm#use-imagemagick-instead-of-gm

It will be a great feature if you add some option to choose GM or IM. Something like this:

```
const map = new StaticMaps({
  width: 800,
  height: 600,
  imageMagick: true
});
```

**Comment by StephanGeorg** [comment]
Hi @saveryanov 

Have you tested this?

**Comment by saveryanov** [comment]
Yes. This works fine for me. I modify your code this way and I don't get any errors. 

```
gm = require('gm').subClass({ imageMagick: true })
```

I think that for the gm module there is no difference in using IM or GM and its documentation has no difference in the methods in this case. I use imagemagick in other projects so this isn't necessary to use extra dependency on graphicsmagick.

**Comment by StephanGeorg** [comment]
Ok, nice. Would you please make a pull request for that feature?

**Comment by saveryanov** [comment]
All done. Check it out:
https://github.com/StephanGeorg/staticmaps/pull/6

**Comment by StephanGeorg** [comment]
Closed w/ #6
